### PR TITLE
Merge compilation instructions from various places

### PR DESCRIPTION
**Pull request by stdedos**
* Rename readme.txt to readme.md to facilitate formatting
* Add repo-combined-folder.png from https://i.imgur.com/EHuyPTJ.png

Signed-off-by: Ntentos Stavros <133706+stdedos@users.noreply.github.com>

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# [processhacker](https://github.com/processhacker)/[plugins-extra](https://github.com/processhacker/plugins-extra)
+
+These are small plugins that are not included with Process Hacker by default.
+
+## Compilation
+_Quoting https://wj32.org/processhacker/forums/viewtopic.php?t=2801#p9194_
```
Comment (dmex) [inline readme.md]: These instructions would be the authoritative source so linking elsewhere like a copy/paste should be removed.

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Your folder should look like this:
+
+![repo-combined-folder](https://raw.githubusercontent.com/processhacker/plugins-extra/master/repo-combined-folder.png)
+
+You can now just open the `plugins-extra\ExtraPlugins.sln` solution just like the `plugins\Plugins.sln` solution. You can also set the default startup project inside Visual Studio and press F5 to run/debug that specific plugin with breakpoints or use the build menu to build all plugins.
+
+> What about a simple possibility to install any of them without installing several gigabytes of Visual Studio?
+
+_Quoting https://github.com/processhacker/processhacker/issues/108#issuecomment-359072916_
```
Comment (dmex) [inline readme.md]: Same here

**Review by dmex — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (stdedos) [reply readme.md]:
```suggestion
```

Changes are easy enough to do, so I'll try to use the new "Suggested Changes" feature!

File: readme.md (hunk 2 above)
Comment (stdedos) [reply readme.md]:
```suggestion
```

Ditto

**Comment by stdedos** [comment]
@dmex Isn't that now to your satisfaction?

**Review by dmex — APPROVED** [review]

**Comment by dmex** [comment]
@stdedos 

The image link is broken? 😄

**Comment by stdedos** [comment]
@dmex no? 😆 

image

**Comment by dmex** [comment]
@stdedos 

It wasn't showing up before... caching maybe 🤔

**Pull request merged**